src/m365/spo/commands/group/group-member-add.spec.ts

```diff
@@ -447,12 +447,12 @@ describe(commands.GROUP_MEMBER_ADD, () => {
         debug: true,
         webUrl: "https://contoso.sharepoint.com/sites/SiteA",
         groupName: "Contoso Site Owners",
-        email: "Alex.Wilber@contoso.com"
+        emails: "Alex.Wilber@contoso.com"
       }
     }), new CommandError(errorMessage));
   });
 
-  it('handles generic error when adding user to a SharePoint Group by groupId and userName', async () => {
+  it('handles generic error when adding user to a SharePoint Group by groupId and userNames', async () => {
```

Comment:
actually, it is a mistake in test description. we are passing `userIds` not `userNames` here.
Do you think we could amend this along the way?